.ci/scripts/test_pybind.py

```diff
@@ -0,0 +1,52 @@
+import torch
```

Comment:
For this test, can we do the following things:

* Add it into the other test modules https://github.com/pytorch/executorch/tree/main/extension/pybindings/test
* Use `unittest` framework
* Add a test for exporting an nn.Module with quantized ops to executorch, read it back using pybind and run it.

Reply:
Ah, thanks

Reply:
Referred to https://github.com/pytorch/executorch/blob/main/exir/tests/test_quant_fusion_pass.py#L40 for quant test.